docs/samples/shards/General/Const/Const.edn

```diff
@@ -0,0 +1,24 @@
+;; declare an int with `Const` and consume in `Math.Multiply`
+999 (Const :Value 2) 
```

Comment:
`999` doesn't impact anything here so it's confusing.

Reply:
Fixed.
commit [#9e2bcf5866b1c0a060fc21bd878baf6dfb84cdbb](https://github.com/fragcolor-xyz/shards/pull/305/commits/9e2bcf5866b1c0a060fc21bd878baf6dfb84cdbb)